sparkle/src/components/PriceTable.tsx

```diff
@@ -20,6 +20,12 @@ const colorTable = {
   emerald: "s-bg-emerald-400",
 };
 
+const textColorTable = {
+  pink: "s-text-pink-900",
+  sky: "s-text-sky-900",
+  emerald: "s-text-emerald-900",
+};
+
```

Comment:
Would pass the lines of the pricing table as an array [{text, icontype(check,cross,minus)}] so that we don't have to bump a new sparkle version when we change the text of a line or add a line